File: Warmup-1/diff21/Diff21.java
package diff21;

public class Diff21 {
    //Retornar a diferença absoluta entre 21 e n, caso n seja maior, retorne o dobro da diferença absoluta

    public int diff21(int n) {
        int diff = 21 - n;
        if (diff < 0){
            diff *= -2;
        }
        return diff;
    }
}

File: Python/Warmup-1/diff21.py
"""
Retorne a diferença absoluta entre entre "n" e 21, exceto quando "n" for maior que 21,
nesse caso retorne o dobro da diferença absoluta.
"""


def diff21(n):
    if n > 21:
        return 2*(n-21)
    else:
        return 21 - n


print(diff21(12))
